### PR TITLE
Update create-api-key.md

### PR DESCRIPTION
**Pull request by edgargarciamacrometa**
I added the complete example for creating an ApiKey

### CODE REVIEW

File: docs/sdks/create-api-key.md [hunk 1]
```diff
@@ -10,12 +10,47 @@ import TabItem from '@theme/TabItem';
 <TabItem value="js" label="Javascript">
 
 ```js
-try{
-await client.createApiKey(keyid);
+const jsc8 = require("jsc8");
+
+// Email and Password to Authenticate client instance
```
Comment (diana-macrometa) [inline docs/sdks/create-api-key.md]:
```suggestion
// Email and password to authenticate client instance
```

File: docs/sdks/create-api-key.md [hunk 2]
```diff
@@ -10,12 +10,47 @@ import TabItem from '@theme/TabItem';
...
+const email = "nemo@nautilus.com";
+const password = "xxxxxx";
+const fabric = "_system";
+const keyid = "id1";
+
+const client = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+// Create an authenticated instance with a JWT token.
```
Comment (diana-macrometa) [inline docs/sdks/create-api-key.md:27]:
```suggestion
// Choose one of the following authentication methods and remove the commenting.

// Create an authenticated instance with a JWT token.
```

File: docs/sdks/create-api-key.md [hunk 3]
```diff
@@ -10,12 +10,47 @@ import TabItem from '@theme/TabItem';
...
 }
-catch(e){
-console.log('API Creation Failed: ', e);
+async function main () {
+  await client
+    .login(email, password)
+    .then((e) => console.log("1. User authentication done!"))
+    .catch((error) => error);
+
+  console.log("2. Creating API Key with KeyID = " + keyid);
```
Comment (diana-macrometa) [inline docs/sdks/create-api-key.md]:
```suggestion
  console.log("2. Creating API key with KeyID = " + keyid);
```

**Review by diana-macrometa — APPROVED** [review]
Looks good, a few minor suggestions.

File: docs/sdks/create-api-key.md [hunk 4]
```diff
@@ -10,12 +10,49 @@ import TabItem from '@theme/TabItem';
...
+
+const client = new jsc8({
+  url: "https://gdn.paas.macrometa.io",
+  fabricName: fabric
+});
+// Choose one of the following authentication methods and remove the commenting.
+
+// Create an authenticated instance with a JWT token.
+// const clientUsingJwt = new jsc8({url: "https://gdn.paas.macrometa.io" , token: "XXXX" , fabricName: fabric});
+// Create an authenticated instance with a API key.
```
Comment (jamesm-macrometa) [inline docs/sdks/create-api-key.md]:
```suggestion
// Create an authenticated instance with an API key.
```

**Review by jamesm-macrometa — APPROVED** [review]
Looks good!  One minor grammar revision

**Pull request merged**